projects/cds/src/services/cds-endpoints.service.spec.ts

```diff
@@ -117,9 +117,11 @@ describe('CdsEndpointsService', () => {
     it('should append query parameters to the URL', () => {
       const FULLY_CALCULATED_URL_WITH_QUERY_PARAMS = `${FULLY_CALCULATED_URL}?site=${BASE_SITE}&language=${LANGUAGE}&pageSize=${PAGE_SIZE}`;
       const STRATEGY_REQUEST: StrategyRequest = {
```

Comment:
this file as well needs to be in lowercase (if the complex object is needed)

Reply:
Done